.github/workflows/whalescale.yml

```diff
@@ -1,25 +0,0 @@
-name: "Check docker image size"
```

Comment:
We don't want to build it twice on github (cost side and time side) so the benefit we get out of it is that minimal that we should delete it and once we have time we should build this for circleci where we already have built the images